ts_src/data_store/cordova_data_store.ts

```diff
@@ -62,16 +112,16 @@ export default class CordovaDataStore implements DataStore {
     return Promise.all([
       new Promise(
         (resolve: (v: number) => void, reject): void => {
-          this.database.transaction((tx: any) => {
-            tx.executeSql(
+          this.database.transaction((db: any) => {
```

Comment:
Since it is reference only, it looks like `database.readTransaction` can be used instead of `database.transaction`.

Reply:
d4c0d6f78b1babda66f1535675c288de3dbe51d3